.github/workflows/release.yml

```
@@ -11,16 +11,12 @@ jobs:
         with:
           fetch-depth: 0
           persist-credentials: false
-      - uses: actions/setup-node@v3
+      - uses: wyvox/action-setup-pnpm@v3
         with:
           node-version: 18
```

```suggestion
    steps:
      - uses: actions/checkout@v5
      - uses: pnpm/action-setup@v4
      - name: Install Node
        uses: actions/setup-node@v6
        with:
          node-version: ${{ env.NODE_VERSION }}
          cache: pnpm
```

Use standard pnpm/action-setup instead of this user-based `wyvox` stuff.